.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -2,13 +2,9 @@
   <profile version="1.0">
     <option name="myName" value="Project Default" />
     <inspection_tool class="FallthruInSwitchStatement" enabled="true" level="WARNING" enabled_by_default="true" />
-    <inspection_tool class="FieldMayBeFinal" enabled="true" level="WARNING" enabled_by_default="true" />
```

Comment:
Why did you remove this inspection profile?

Reply:
I reenabled the test class naming convention inspection because the pattern was updated to allow ...IT. Then IntelliJ also removed the final field inspection. Turns out it is now enabled by default.
Would be great if the default IntelliJ inspection profiles would be versioned but they aren't.